application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -536,6 +550,39 @@ type FeedPublisher {
   url: String!
 }
 
+"""
+Exact dated stoptime represents the time when a specific trip on a specific date arrives to and/or departs from a specific stop.
+This can include realtime estimates.
+"""
+type FixedStopTimeOnServiceDate {
+  "Scheduled arrival time to the stop and a realtime estimate, if one exists."
+  arrival: ArrivalDepartureTime
+  "Scheduled departure time from the stop and a realtime estimate, if one exists."
+  departure: ArrivalDepartureTime
+  "The stop where this arrival/departure happens"
+  stop: Stop
+}
+
+"A fixed (i.e. not flexible or frequency based) trip on a specific service date"
+type FixedTripOnServiceDate {
+  """
+  The service date when the trip occurs.
+    
+  **Note**: A service date is a technical term useful for transit planning purposes and might not
+    correspond to a how a passenger thinks of a calendar date. For example, a night bus running
+    on Sunday morning at 1am to 3am, might have the previous Saturday's service date.
+  """
+  date: LocalDate!
```

Comment:
Presumably this isn't called serviceDate because it's already in the type name.

Reply:
Either is ok for me.

Reply:
The documentation is very clear what it does, so I'm also fine with both. But since we want to use a ubiquitous language, I vote for serviceDate. @t2gran can give his opinion.

Reply:
We decided to call it serviceDate